src/main/resources/templates/summary.html

```diff
@@ -20,6 +20,16 @@ <h2 th:text="${survey.getTitle()}"></h2>
                         <li th:each="option : ${question.options}" th:text="${option}"></li>
                     </ul>
                 </div>
+
+                <!-- Display responses if the question is a CommentQuestion -->
+                <div th:if="${#strings.equals(question.class.simpleName, 'CommentQuestion')}">
+                    <h4>Responses:</h4>
+                    <ul>
+                        <li th:each="response : ${question.commentResponses}" th:text="${response.text}">
```

Comment:
Do you think doing this would work? Do we need the list of comment responses?

```suggestion
                        <li th:each="response : ${question.responses}" th:text="${response.text}">
```